pwndbg/elf.py

```diff
@@ -285,3 +285,27 @@ def map_inner(ei_class, ehdr, objfile):
         page.objfile = objfile
 
     return tuple(sorted(pages))
+
+
+def getjmpslots(path):
+    """
+    Use the readelf command with the parameter -r
+    in order to retrieve the jump slots inside the 
+    binary specified in path.
+    """
+    program = pwndbg.which.which("readelf")
+    if program:
+        argv = [program, "-r", path]
+        readelf_out = subprocess.check_output(argv).decode('utf-8')
+        if "Error" in readelf_out:
```

Comment:
It would be better to check for return code instead of a string content. Imagine having a function/relocation called `"Error"` ;).

E.g.:
```bash
[dc@dc:~]$ readelf -r a
readelf: Error: 'a': No such file
[dc@dc:~]$ echo $?
1
[dc@dc:~]$ readelf -r /bin/ls 1>/dev/null
[dc@dc:~]$ echo $?
0
```